packages/inventory/config/webpack.constants.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
     externals: [
-        /^@patternfly\/.*/,
+        /^@patternfly\/react-core.*/,
+        /^@patternfly\/react-icons.*/,
```

Comment:
i feel like this `/^@patternfly\/react-icons.*/` should be like this `/^@patternfly\/react-icons/*`